CrossPlatform/BasePage.cs

```diff
@@ -16,18 +16,16 @@ public abstract class BasePage
 
         protected abstract Trait Trait { get; }
 
-        protected Func<AppQuery, AppQuery> CurrentTrait { get; private set; }
-
         protected BasePage()
         {
             app = AppInitializer.App;
 
             OnAndroid = app.GetType() == typeof(AndroidApp);
             OniOS = app.GetType() == typeof(iOSApp);
 
-            CurrentTrait = OnAndroid ? Trait.AndroidTrait : Trait.iOSTrait;
+            Trait.Platform = OnAndroid ? Platform.Android : Platform.iOS;
```

Comment:
What do we think of storing the current platform in the `AppInitializer` like we do with `app`? This would simplify these three lines.